<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INSUPLTQAPT" failures="1" tests="1" name="testngframeworkbasics.Test_Case_3_SoftAssertions" time="0.007" errors="0" timestamp="2025-03-27T00:23:21 IST" skipped="0">
  <testcase classname="testngframeworkbasics.Test_Case_3_SoftAssertions" name="validateImageUNTitle" time="0.007">
    <failure message="The following asserts failed:
	Image Mismatched expected [true] but found [false],
	UN Mismatched expected [true] but found [false],
	Title Mismatched expected [Webuy.com] but found [Google.com]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Image Mismatched expected [true] but found [false],
UN Mismatched expected [true] but found [false],
Title Mismatched expected [Webuy.com] but found [Google.com]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at testngframeworkbasics.Test_Case_3_SoftAssertions.validateImageUNTitle(Test_Case_3_SoftAssertions.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1207)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- validateImageUNTitle -->
  <system-out>
    <![CDATA[<a href="C:\Users\pthakare\Selenium Testing\eclipse-workspace\SeleniumTesting\screenshots\page.png" target="_blank">Screenshot Captured !!!</a>
<br>
<a href="C:\Users\pthakare\Selenium Testing\eclipse-workspace\SeleniumTesting\screenshots\page.png" target="_blank"><img height=200 width=200 src="C:\Users\pthakare\Selenium Testing\eclipse-workspace\SeleniumTesting\screenshots\page.png"></a>]]>
  </system-out>
</testsuite> <!-- testngframeworkbasics.Test_Case_3_SoftAssertions -->
